ApleStore/ViewController/WebPageViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  WebPageViewController.swift
+//  ApleStore
+//
+//  Created by Timofey Privalov on 14.07.2021.
+//
+
+import UIKit
+import WebKit
+
+final class WebPageViewController: UIViewController, WKUIDelegate {
```

Comment:
```suggestion
final class WebPageViewController: UIViewController {
```
 WKUIDelegate - в extension